ios/BT/bridge/ExposureEventEmitter.m

```diff
@@ -2,13 +2,18 @@
 #import <React/RCTEventEmitter.h>
 
 // Notification/Event Names
-NSString *const onExposuresChanged = @"EXPOSURES_CHANGED";
+NSString *const onExposuresChanged = @"onExposureRecordUpdated";
 
 @interface ExposureEventEmitter : RCTEventEmitter <RCTBridgeModule>
 @end
 
 @implementation ExposureEventEmitter
 
++ (BOOL)requiresMainQueueSetup
```

Comment:
@mattThousand , wanted to point this out in particular, is this okay to be running on the main thread?

Reply:
chatted irl, theres no need fo this to be on the main thread since its not updating ui, but it is fine to keep on as well.